src/lib/queries/exchangeRates.ts

```
@@ -124,62 +162,53 @@ export function resolveRatio(
 	return Number.isFinite(parsed) && parsed > 0 ? parsed : 1;
 }
 
-/**
- * TanStack-shaped query so existing consumers (`$exchangeRatesQuery.data`)
- * keep working when the live API replaces the fixture. The queryFn is
- * synchronous-with-Promise-wrap — there's no network call.
- */
-export function createExchangeRatesQuery() {
+function warnWrapRatioErrors(response: ApiWrapRatiosResponse): void {
+	if (response.errors?.length) {
+		console.warn('[wrap-ratio] REST API returned partial errors', response.errors);
```

ESLint

[error] Unexpected console statement.

(no-console)